CHANGELOG.md

```diff
@@ -331,6 +331,8 @@
 - [Updated `rename_columns` to new API. Added `first_row`, `second_row` and
   `last_row` to Table types][5719]
 - [Aligned names of columns created by column operations.][5850]
+- [Improved `cross_tab`. Renamed `fill_missing` and `is_missing` to
+  `fill_nothing` and `is_nothing`. Added `fill_empty`.][5863]
```

Comment:
missing link to 5863?

Reply:
Oh, thanks I have completely forgot! I haven't been adding stuff to the changelog for a while.